keyboards/40percentclub/polyandry/readme.md

```diff
@@ -0,0 +1,45 @@
+# Polyandry/polypad
+
+![Polyandry](https://1.bp.blogspot.com/-97uoICZvZec/XSdpvCVFFlI/AAAAAAACVZY/BlCRjxjDqDAxOZa6C5ddQr0OsFQ9VZv5wCLcBGAs/s1600/d1.jpg)
+===
+
+3x4/4x3 grid layout, multiple controllers supported
+- Pro Micro
+- PJRC Teensy 2.0
+
+(**The below are not currently ported:**)
+
+- PJRC Teensy LC/3.2 
+- PJRC Teensy 3.5/3.6
+- Adafruit ItsyBitsy ATMega32u4
+- Adafruit ItsyBitsy M0
+- Adafruit ItsyBitsy M4
+
+The matrix is electrically 1x12 with no diodes, with a thirteenth pin used for the column rather than a ground.
+
+The default keymap is a testing keymap that is intended to make sure all of the keys on the matrix are working and properly assigned.
+
+[The original (minimal arduino sketch) firmware, with support for some unsupported controllers](https://git.40percent.club/di0ib/Misc/src/branch/master/polyandry)
+
+Keyboard Maintainer: QMK Community  
+Hardware Supported: Polyandry PCB  
+Hardware Availability: [Polyandry gerbers are on 40percent.club](https://www.40percent.club/2019/07/polyandry.html)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```